charts/restore/values.yaml

```diff
@@ -11,13 +11,20 @@ cassandraDatacenter:
   # restored
   name: dc1
 
-# -- In-place restore will restore the backup to the source cluster. Note that
-# this will trigger a rolling restart of the cluster.
+# -- In-place restore will restore the backup to the source cluster.
 inPlace: true
 
 # -- When true will shutdown the entire Cassandra cluster. The underlying
 # StatefulSets are scaled down to zero. Persistent volumes remain intact. If the
-# backup includes schema changes like dropping a table, then set this to true;
-# otherwise, the changes will be lost via gossip from nodes that have not yet
-# been restored.
-shutdown: false
+# backup includes schema changes like dropping a table, then must be set to
+# true; otherwise, the changes will be lost via gossip from nodes that have
+# not yet been restored. It is recommended in general to shutdown the cluster
+# prior to a restore to avoid data inconsistencies and/or data loss that could
+# happen with clients writing to the cluster while the restore operation is in
+# progress.
+#
+# When set the cluster is shutdown the restore operations happen in parallel
```

Comment:
nit: add the word `and`

_suggested change_:
```
When set the cluster is shutdown and the restore operations happen in parallel
```

Reply:
Yup, I want to address it :)